### PR TITLE
fix CubicLineChartActivity, BarLineChartTouchListener

### PR DESCRIPTION
**Pull request by ggggxiaolong**

### CODE REVIEW

**Comment by ggggxiaolong** [comment]
1. Demo CubicLineChartActivity increase count more than 100 wiil crash
2. Demo PerformanceLineChart scale x will case OOM

**Comment by hannesa2** [comment]
> 1. Demo CubicLineChartActivity increase count more than 100 will crash
> 
>     2. Demo PerformanceLineChart scale x will case OOM

Thank you, I will investigate

File: MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/CubicLineChartActivity.java [hunk 1]
```diff
@@ -105,7 +105,7 @@ protected void onCreate(Bundle savedInstanceState) {
     private void setData(int count, float range) {
 
         ArrayList<Entry> values = new ArrayList<>();
-        Double[] sampleValues = DataTools.Companion.getValues(100);
+        Double[] sampleValues = DataTools.Companion.getMuchValues(count);
```
Comment (hannesa2) [inline MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/CubicLineChartActivity.java:108]: > 1. Demo CubicLineChartActivity increase count more than 100 will crash

Is this already the fix for this ?

Comment (hannesa2) [reply MPChartExample/src/main/java/com/xxmassdeveloper/mpchartexample/CubicLineChartActivity.java:108]: I see it fixes it. Thank you

**Comment by hannesa2** [comment]
> 2\. Demo PerformanceLineChart scale x will case OOM

I can't reproduce the error. Maybe can you show me how to do so ?
image

**Pull request merged**